recipes/platform.hashing/all/conanfile.py

```diff
@@ -47,6 +47,10 @@ def _compilers_minimum_version(self):
     def _minimum_cpp_standard(self):
         return 20
 
+    def requirements(self):
+    		if Version(self.version) >= "0.3.0":
```

Comment:
Why did https://github.com/conan-io/conan-center-index/pull/11490 pass?

Reply:
I have the same question @prince-chrismc 🌚

Reply:
I don't know, but please, do not mix space and tabs. Usually, python is sensitive to it.

Reply:
nothing obvious in the code shows where this is being used 🙈 https://github.com/linksplatform/Hashing/compare/0.2.0...cpp_0.3.0

maybe the test package needs a little help